src/main/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/service/cas1/Cas1ApplicationDomainEventService.kt

```diff
@@ -114,6 +114,7 @@ class Cas1ApplicationDomainEventService(
         metadata = mapOf(
           MetaDataName.CAS1_APP_REASON_FOR_SHORT_NOTICE to submitApplication.reasonForShortNotice,
           MetaDataName.CAS1_APP_REASON_FOR_SHORT_NOTICE_OTHER to submitApplication.reasonForShortNoticeOther,
+          MetaDataName.CAS1_REQUESTED_AP_TYPE to application.apType.name,
```

Comment:
This was really an oversight on my part when i added the previous two metadata fields here, but would you mind updating the unit test to check these 3 metadata fields as you've done with the assessment domain event?